src/main/java/com/moviesuggestion/HelloWorld.java

```diff
@@ -0,0 +1,12 @@
+package com.moviesuggestion;
```

Comment:
I guess you could remove this class now since you already have a starting class.